src/3D/MeshPack.h

```diff
@@ -64,6 +64,7 @@ class MeshPack
 	void loadTextures(const std::map<std::string, pack::G3DTexture>& textures);
 	void loadMeshes(const std::vector<std::vector<uint8_t>>& meshes);
 
+	bool _enableUnknownMeshes = false;
```

Comment:
Could you move default values to the constructor to be more consistent?

Reply:
Done. Updated the constructor